core/module.go

```diff
@@ -122,11 +123,25 @@ func (mod *Module) IDModule() *call.Module {
 	return call.NewModule(mod.InstanceID, mod.Name(), ref)
 }
 
+func (mod *Module) addCompatToCtx(ctx context.Context) (context.Context, error) {
```

Comment:
I *think* this isn't needed? By calling this in the server, doesn't this propagate down everywhere to where it needs to go?

Might be just leftover, but can't quite tell.